templates/vhost/_ssl.erb

```diff
@@ -36,6 +34,8 @@
   <%- if @ssl_crl_check && scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
   SSLCARevocationCheck    "<%= @ssl_crl_check %>"
   <%- end -%>
+  <%- if @ssl_verify_client -%>
+  SSLVerifyClient         <%= @ssl_verify_client %>
```

Comment:
Could you explain why this was moved down? The commit message says something about `SSLCACertificateFile` but I don't understand the purpose of moving it.

Reply:
It's deliberate; For client verification to occur, at least the SSLCACertificateFile needs to be specified (which may not have been specified if rest of VHost doesn't use client authentication).

In other words, I can create a VHost which presents a server certificate but then on a particular Location directive, further require a client certificate. In that case, I will need to also resolve a SSLCACertificateFile in order to validate the client certificate.